### DISCUSSION TITLE
How do you convert the prompt scheduler inputs ?

### DISCUSSION THREAD

**Discussion by JoeTheDino** [FizzleDorf/ComfyUI_FizzNodes#90] [Q&A] [answered]
Hello, 
It's been a while i have been searching how to convert the prompt scheduler inputs like on the other nodes ?
The option is not there.
![image_2024-05-14_111910046](https://github.com/FizzleDorf/ComfyUI_FizzNodes/assets/16972431/ea1d92aa-d018-4708-9357-4ca2ac148300)
Thank you very much

**Comment by FizzleDorf** [comment, marked as answer]
hey @JoeTheDino! The inputs that can't be changed were set to forced to reduce clutter. The readme needs to be updated to reflect this. Is there a specific case you need them to be widgets?

**Reply by JoeTheDino** [reply]
Hi @FizzleDorf ! Thanks for your reply and for the amazing work. I don't have any use case in particular but searching for it drove me a bit crazy. 😄
